src/lua/utils.c

```diff
@@ -53,6 +53,37 @@ uint32_t CTID_UUID;
 uint32_t CTID_DATETIME = 0;
 uint32_t CTID_INTERVAL = 0;
 
+/** A copy of index2adr() from luajit/src/lj_api.c */
+static TValue *
+luaL_index2adr(lua_State *L, int idx)
```

Comment:
Generally, we try to avoid using `luaL` prefix in function naming (see #4577 for more info), but this still respects our current guidelines and this function is not exported, so can be freely renamed later. Feel free to ignore.

Reply:
Renamed to `index2adr()`.